spec/lib/thinkst_canary/token/pdf_acrobat_reader_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require 'lib/thinkst_canary/token/shared_examples'
+
+RSpec.describe ThinkstCanary::Token::PdfAcrobatReader do
+  include_examples 'a Canary token', 'pdf-acrobat-reader' do
+    let(:file_upload) { instance_double(Faraday::UploadIO) }
+    let(:extra_token_options) { { file: StringIO.new } }
+    let(:extra_request_options) { { pdf: file_upload } }
+
+    before { allow(Faraday::UploadIO).to receive(:new).and_return(file_upload) }
```

Comment:
do these belong here or in the shared example? They are repeated for the `doc-msword` version implying they could reside in the shared example - only the key `pdf:` is different